src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -1322,4 +1318,23 @@ public Response startQuartzJobService(@Context final HttpServletRequest request)
             return new SegueErrorResponse(Status.INTERNAL_SERVER_ERROR, e.getMessage()).toResponse();
         }
     }
+
+    @GET
+    @Path("/misuse_stats/{agent_identifier}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get the site misuse stats for a given user.")
```

Comment:
Ahhh I see, since all relevant misuse info for a user might be stored under a few different agent identifiers it does make this feature a bit incomplete and not as useful as it could be. 
I'll rewrite it as "top-N keys per event label", which would make less clutter in the front-end user manager as well as being more useful 👍